### ISSUE TITLE
Removes  "<ns0: " from xml result

### ISSUE THREAD

**Issue by pceglie** [microsoft/azuredatastudio#25220]
- Azure Data Studio Version:
- OS Version:

Steps to Reproduce:

1. SELECT from a table with an xml field
![image](https://github.com/microsoft/azuredatastudio/assets/128714414/21782e95-f3b0-4fc5-b4b1-d9eb17c9ba3b)

2. Click on xml cell. As you can see the firts part was removed
![image](https://github.com/microsoft/azuredatastudio/assets/128714414/22ee89ad-a519-4054-942d-ffa3687f17b8)

SQL Server Management Studio is not affected from that issue.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by cheenamalhotra** [comment]
Can confirm this occurs as far as 1.41.. needs RCA

**Comment by pceglie** [comment]
Yes, as you can see on top of my post the version is the last (1.47.1).
Regards

**Comment by Sharparam** [comment]
Still an issue on version 1.51.0.

![Image](https://github.com/user-attachments/assets/30327ab9-955b-495d-955d-c53a3bae0f18)
